index.bs

```diff
@@ -334,6 +334,29 @@ partial interface NeuralNetworkContext {
 };
 </script>
 
+### reshape ### {#api-neuralnetworkcontext-reshape}
+<script type=idl>
+partial interface NeuralNetworkContext {
+  Operand reshape(Operand input, sequence<long> newShape);
+};
+</script>
+
+<div algorithm=reshape>
+    **Arguments:**
+        - *input*: an {{Operand}}. The input N-D tensor.
+        - *newShape*: a sequence of {{long}}. The logical shape of the output
+            tensor. The number of elements implied by *newShape* must be the
+            same as the number of elements in the input tensor. At most one
+            dimension of the *newShape* can be -1. In this case, the size of
+            that dimension is computed so that the total size remains constant.
+
+    **Returns:**: an {{Operand}}. The output N-D tensor. The values of the
+    output tensor are the same as values of the input tensor. The logical shape
+    of the output tensor is specified by the *newShape* argument.
+
+    Reshapes a tensor.
```

Comment:
"Reshape a tensor to a given new shape."

Reply:
Sounds better. I'll incorporate it. Thanks!

Reply:
fixed in the new commit. please take a look.